docs/to_clarify.md

```diff
@@ -0,0 +1,8 @@
+## Question
+1. What are the existing solutions that was mentioned in point 2 of the "Possible Use Cases"
+
+1. What are the digital solutions that the industry have tried? To potentially prevent designing similar solutions 
+
+1. Is there any other activities that depend on the current check-in/out system. In the case when solution replaces current check-in/out systems do we need to cater for these dependencies? For example, if a check-in kiosk is currently used, and it has maps to guide guests after checking in, this would mean our solution would also need to include guiding guests after checking in 
```

Comment:
Good question. Again, we could be more specific with the words we use: Are there **systems** depending on the existing check-in/out system? If so... (continue yourself haha).

Also, this question is somewhat linked to what @alvinlim540 mentioned concerning Booking Engine and the ease of integration; whether there's documentation & assistance will be provided to us for integration. It should be consolidated as 1 big point.

Reply:
Side note, but on the same point of Integration. Based on the challenge brief, under the header "What are we looking for":

> As the challenge is hosted by two hotels, the solution must be able to integrate with both digital concierge systems

We should ask about their existing Digital Concierge and possibly try it out to have a look and feel. Or perhaps, we can try Googling or looking up App Store to see if we can find anything online. Can someone look into this? @alvinlim540 @ahyong92